avm/res/network/private-endpoint/main.bicep

```diff
@@ -219,10 +219,10 @@ type ipConfigurationsType = {
 
   @description('Required. Properties of private endpoint IP configurations.')
   properties: {
-    @description('Required. The ID of a group obtained from the remote resource that this private endpoint should connect to.')
+    @description('Required. The ID of a group obtained from the remote resource that this private endpoint should connect to. If used with private link service connection, this property must be defined as empty string.')
     groupId: string
```

Comment:
Hey @segraef, I'm curious if it would also have worked with a nullable parameter instead. Or was the reason to not make it nullable but use an empty string to treat the groupId still as required?

Reply:
I looked at this too @AlexanderSehr, but don't think making it nullable is practical because when you use the snippet "required-properties" it wont show up. And besides that I think in most situations you need to use groupId, so having that property as required is better.

Reply:
If that was the reason, fair :) Just wanted to double-check if it was intentional. Using the empty string as a backdoor is a lot more custom than the nullable path, so it should be a concious decision

Reply:
It's optional but required 🍰